Gemfile

```diff
@@ -22,6 +22,7 @@ gem 'turbolinks'
 gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
+gem 'httparty'
```

Comment:
This is the library we're going to use to snag API information. It's quick and dirty.